tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ts

```diff
@@ -56,6 +62,40 @@ const DAY_IN_MS = 24 * 1000 * 60 * 60;
         </g>
       </svg>
     </div>
+    <mat-menu
+      #manualControl="matMenu"
+      xPosition="before"
+      [yPosition]="axis === 'y' ? 'above' : 'below'"
+    >
```

Comment:
> If the mat-menu is what's handling the dismissal on click/keydown, can we remove all the stopPropagation's from the div/buttons inside, and instead move those event listeners to the <mat-menu> itself?

Does not seem to work. Specifically, mat-menu uses portal and content inside the mat-menu gets projected onto the overlay, not the `mat-menu` itself. We can put a wrapper around all of our element but I think that defeats keyboard navigation or some of the assumption of the mat-menu. Will keep it as is.

> If Cancel/Change could call something like menu.close, it would be more explicit.

At least for testing, added that explicit close.

> Also, it seems click(+drag) on the mat-menu itself will dismiss it, when it should not.

Might be annoying but it is okay as is.